apps/proxy-auth/src/app/otp/component/login/login.component.scss

```diff
@@ -2,9 +2,49 @@
     display: flex;
     flex-direction: column;
     gap: 16px;
+    font-size: medium;
+    font-weight: 600;
+    a {
+        font-size: medium;
+        font-weight: 400;
+    }
 }
 .error-message {
     font-size: 14px;
     color: #f44336;
     margin-bottom: 16px;
 }
+.close-dialog {
```

Comment:
You can use same css for class `.close-dialog`, `back-button` and add new class for `right` and `left`